src/main/java/Sidtacphi.java

```diff
@@ -0,0 +1,255 @@
+                if (Objects.equals(input, "bye")) {
+                    stopBot();
+                    break;
+                } else if (Objects.equals(input, "list")) {
+                    showTaskList();
+                } else if (input.startsWith("mark")) {
+                    markTaskAs(true, input);
+                } else if (input.startsWith("unmark")) {
+                    markTaskAs(false, input);
+                } else if (input.startsWith("todo")) {
+                    addTask(TaskType.TODO, input);
+                } else if (input.startsWith("event")) {
+                    addTask(TaskType.EVENT, input);
+                } else if (input.startsWith("deadline")) {
+                    addTask(TaskType.DEADLINE, input);
+                } else if (input.startsWith("delete")) {
+                    deleteTask(input);
+                } else {
+                    throw new SidException("\"" + input + "\" is not a valid command.");
+                }
+                System.out.print("\nYou: ");
+            } catch (SidException e) {
+                    System.out.print("\n");
+                    System.out.println(e.getMessage());
+                    System.out.print("\nYou: ");
+            }
+        }
+        scan.close();
+    }
+
+    /**
+     * Adds the input to the task_list kept track of by the bot.
+     * 
+     * @param taskType type of task to add
+     * @param input input to add to the task_list kept by the bot
+     */
+    private static void addTask(TaskType taskType, String input) throws SidException {
+        String[] inputArgs;
+        switch (taskType) {
+            case TODO:
+                if (input.length() < 5) {
+                    throw new SidInvalidFormatException("Please input a name for your Todo task.");
+                } else if (input.charAt(4) == ' ') {
+                    taskList.add(new Todo(input.substring(5)));
+                } else {
+                    throw new SidException("\"" + input + "\" is not a valid command.");
+                }
+                break;
+            case EVENT:
+                if (input.length() < 6) {
+                    throw new SidInvalidFormatException("Please input a name for your Event" 
+                    + "task, along with a start and end time.");
+                } else if (input.charAt(5) == ' ') {
+                    inputArgs = input.substring(6).split("\\s*/from\\s*");
+                    if (inputArgs.length == 2) { 
+                        String[] startAndEnd = inputArgs[1].split("\\s*/to\\s*");
+                        if (startAndEnd.length == 2) {
+                            taskList.add(new Event(inputArgs[0], startAndEnd[0], startAndEnd[1]));
+                        } else {
+                            throw new SidInvalidFormatException("Please put in the starting and ending time " 
+                            + "using \"/from <time>\" followed by \"/to <time>\" for Event tasks.");
```

Comment:
Good job wrapping long lines of code. But, I think it is recommended that indentation for wrapped lines should be 8 spaces.

Reply:
yuppp